### DISCUSSION TITLE
Why do I need to use the host network

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#4821] [Q&A] [answered]
When I need to use DHCP service, why do I need to set `--network` to `host`?

![image](https://user-images.githubusercontent.com/51149477/183295160-0b0e46a9-ecee-48f3-af87-1232f48f0397.png)

**Comment by centralhardware** [comment, marked as answer]
because DHCP using broadcast so I think DHCP don't working with docker container network isolation
